test/integration/backupbucket/backupbucket_test.go

```diff
@@ -111,11 +109,8 @@ var _ = BeforeSuite(func() {
 		By("deleting aws provider secret")
 		deleteBackupBucketSecret(tc.ctx, tc.client, tc.secret)
 
-		By("deleting namespaces")
+		By("deleting test namespace")
 		deleteNamespace(tc.ctx, tc.client, tc.testNamespace)
-		if !tc.gardenNamespaceExists {
-			deleteNamespace(tc.ctx, tc.client, tc.gardenNamespace)
```

Comment:
I guess also the `gardenNamespace` is not needed.

Reply:
good catch, thanks!